addons/godot-next/2d/vector_display_2d.gd

```diff
@@ -0,0 +1,49 @@
+tool
+class_name VectorDisplay2D
+extends Node
+# Displays Vector2 members in the editor via Position2D nodes.
+
+export(String) var variable_name
+export(bool) var relative = true
+
+var _storage: Node2D
+
+
+func _process(delta):
+	if not variable_name:
+		printerr("VectorDisplay2D: Please provide a variable name.")
+		return
```

Comment:
Seems weird that we'd need to print this error message every frame, potentially. Should this be moved to a `_get_configuration_warning()` and/or to an `_ready()` or something so it only fires once?

Reply:
Also, if we do print anything in script code, we should favor `push_error()` here so that errors are shown in the editor console, yes?

Reply:
Yeah, good point. I pushed a change which simply stores the old value, and then compares against it. It only prints the warning if the name changes.